Cargo.toml

```diff
@@ -150,7 +150,7 @@ rowan = { version = "0.16.1" }
 rustybuzz = { version = "0.20.0" }
 send_wrapper = { version = "0.6.0" }
 serde = { version = "1.0.163", default-features = false, features = ["derive"] }
-serde_json = { version = "1.0.96" }
+serde_json = { version = "1.0.96", default-features = false }
```

Comment:
If your enabling the default features everywhere it is used, this should probably be better to keep it unchanged. (enable the default feature by default)